packages/salesforcedx-vscode-core/src/commands/isvdebugging/bootstrapCmd.ts

```diff
@@ -64,52 +78,278 @@ export class IsvDebugBootstrapExecutor extends SfdxCommandletExecutor<{}> {
       .build();
   }
 
-  public execute(response: ContinueResponse<IsvDebugBootstrapConfig>): void {
+  public buildRetrieveOrgSourceCommand(data: IsvDebugBootstrapConfig): Command {
+    return new SfdxCommandBuilder()
+      .withDescription(
+        nls.localize('isv_debug_bootstrap_step3_retrieve_org_source')
+      )
+      .withArg('force:mdapi:retrieve')
+      .withFlag('-r', this.relativeMetdataTempPath)
```

Comment:
Let's use the full parameter names so it's easier to read. Other commands follow that pattern.